tests/test_analyzer.py

```diff
@@ -0,0 +1,116 @@
+# Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest.mock import patch
+from model_analyzer.analyzer import Analyzer
+from model_analyzer.cli.cli import CLI
+from model_analyzer.config.input.config_command_profile import ConfigCommandProfile
+from model_analyzer.result.model_result import ModelResult
+from model_analyzer.state.analyzer_state_manager import AnalyzerStateManager
+from model_analyzer.triton.model.model_config import ModelConfig
+
+from google.protobuf import json_format
+from tritonclient.grpc import model_config_pb2
+
+from .common import test_result_collector as trc
+
+from .mocks.mock_config import MockConfig
+
+
+class TestAnalyzer(trc.TestResultCollector):
+    """
+    Tests the methods of the Analyzer class
+    """
+
+    def _evaluate_config(self, args, yaml_content):
```

Comment:
reorder the file so the tests are at the top and helper functions at the bottom

Reply:
reordered 👍